engine/class_modules/sc_warrior.cpp

```diff
@@ -6825,7 +6854,9 @@ void warrior_t::apl_prot()
     default_list -> add_action( racial_actions[ i ] );
 
   default_list -> add_action( "potion,if=buff.avatar.up|target.time_to_die<25" );
-  default_list -> add_action( this, "Ignore Pain","if=buff.ignore_pain.down&rage>50");
+  default_list -> add_action( this, "Ignore Pain", "if=target.health.pct>20&!covenant.venthyr,line_cd=15" ); //Prioritize Execute over Ignore Pain as a rage dump below 20% 
+  default_list -> add_action( this, "Ignore Pain", "if=target.health.pct>20&target.health.pct<80&covenant.venthyr,line_cd=15"); //Venthyr Condemn has 2 execute windows, 20% and 80% 
```

Comment:
I meant APL comments that show up in the generated version: 

```suggestion
  default_list -> add_action( this, "Ignore Pain", "if=target.health.pct>20&!covenant.venthyr,line_cd=15", "Prioritize Execute over Ignore Pain as a rage dump below 20%" );
  default_list -> add_action( this, "Ignore Pain", "if=target.health.pct>20&target.health.pct<80&covenant.venthyr,line_cd=15", "Venthyr Condemn has 2 execute windows, 20% and 80%" );
```